packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -78,13 +86,43 @@ $mdc-toolbar-mobile-breakpoint: 599px;
   &__title {
     @include mdc-typography(title);
 
+    align-self: center;
     margin: 0;
+    margin-left: 16px;
+    padding: $mdc-toobar-element-vertical-padding 0;
     line-height: 1.5rem;
     text-overflow: ellipsis;
     white-space: nowrap;
     overflow: hidden;
     z-index: 1;
   }
+
+  &__icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: $mdc-toobar-element-vertical-padding 8px;
+    border: none;
+
+    @include mdc-theme-prop(color, text-primary-on-primary);
+  }
+
+  &__icon--menu {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: $mdc-toobar-element-vertical-padding;
+    border: none;
+
+    @include mdc-theme-prop(color, text-primary-on-primary);
+  }
+
+  &__icon__button {
```

Comment:
I took at look at the case when you use <a> link.
There are currently 2 style overwrite, one in `demo.css`, the other one in `default-demo.html`. Both of them is custom overwrite but not part of our `mdc-toolbar.scss`.
Rather than having another `&__icon__link` and based on my previous suggestion of `menu` and `icon`, how about merge the overwrite of both `button` and `link` all as part of `__icon`?